packages/studio-ui/src/web/components/Layout/Toolbar/index.tsx

```diff
@@ -28,15 +28,7 @@ const Toolbar: FC<Props> = (props) => {
   return (
     <header className={style.toolbar}>
       <div className={style.list}>
-        {window.IS_PRO_ENABLED ? (
-          <span />
-        ) : (
-          <Tooltip position="right-bottom" content={lang.tr('studio.flow.toolbar.salesCallToActionDescription')}>
-            <a className={style.cta_btn} target="_blank" href="https://botpress.com/request-trial-from-app">
-              {lang.tr('studio.flow.toolbar.salesCallToAction')}
-            </a>
-          </Tooltip>
-        )}
+        <span />
```

Comment:
why keep an empty span ?

Reply:
Because of the alignment (space-between). Didn't want to play with the style, and if we add stuff in the future, it will be ready